Readme.md

```diff
@@ -1 +1,46 @@
-# ROI Hunter Easy
\ No newline at end of file
+# ROI Hunter Easy
+
+## Installation
+- navigate to **release** tab in our GitHub repository
```

Comment:
Can be replaced by the link https://github.com/roihunter/roi-hunter-easy-presta-plugin/releases/latest